packages/synapse-react-client/src/components/DownloadCart/DownloadListTable.test.tsx

```diff
@@ -74,59 +89,18 @@ function renderComponent() {
 describe('DownloadListTable tests', () => {
   beforeEach(() => {
     jest.clearAllMocks()
-    mockUseGetAvailableFilesToDownloadInfinite.mockReturnValue({
-      data: {
-        pages: [
-          {
-            page: page1,
-            nextPageToken: '50a0',
-            concreteType:
-              'org.sagebionetworks.repo.model.download.AvailableFilesResponse',
-          },
-          {
-            page: page2,
-            nextPageToken: undefined,
-            concreteType:
-              'org.sagebionetworks.repo.model.download.AvailableFilesResponse',
-          },
-        ],
-        pageParams: [],
-      },
-      fetchNextPage: mockFetchNextPage,
-      hasNextPage: false,
-      isLoading: false,
-      isSuccess: true,
-      error: null,
-      isError: false,
-      isIdle: false,
-      isLoadingError: false,
-      isRefetchError: false,
-      status: 'success',
-      fetchPreviousPage: jest.fn(),
-      isFetchingNextPage: false,
-      isFetchingPreviousPage: false,
-      dataUpdatedAt: 0,
-      errorUpdatedAt: 0,
-      failureCount: 0,
-      errorUpdateCount: 0,
-      isFetched: false,
-      isFetchedAfterMount: false,
-      isFetching: false,
-      isPlaceholderData: false,
-      isPreviousData: false,
-      isRefetching: false,
-      isStale: false,
-      refetch: jest.fn(),
-      remove: jest.fn(),
-    })
   })
   it('loads more available download files when inView', async () => {
-    mockAllIsIntersecting(true)
-
     renderComponent()
 
     const fileEntity1 = await screen.findAllByText('file1.txt')
     expect(fileEntity1).toHaveLength(1)
+
+    // trigger fetching page 2
+    act(() => {
+      mockAllIsIntersecting(true)
+    })
```

Comment:
Procedurally load the 2nd page, just as a user would. Wrap in `act` since the intersection observer status will trigger a component update.